geps/gep-713/index.md

```diff
@@ -3,961 +3,927 @@
-  Direct Attached Policies (or more simply, Direct Policies.)
-
-The biggest difference between the two types of Policy is that Direct Attached
-Policies are a strict subset of Policy objects with criteria designed to make
-it _much_ easier to understand the state of the system, and so are simpler to
-use and can use a more simple `status` design.
-
-However, Inherited Policies, because of the nature of the useful feature of having
-settings cascade across multiple objects in a hierarchy, require knowledge of
-more resources, and are consequently harder to understand and require a more
-complex status design.
-
-Splitting these two design patterns apart into separate GEPs is intended to
-allow proceeding with stabilizing the simpler (Direct) case while we work on
-solving the status problem for the more complex (Inherited) case.
-
-Direct Attached Policies are further specified in the addendum GEP GEP-2648,
-Direct Policy Attachment.
+(See status definitions [here](/geps/overview/#gep-states))
 
-Inherited Policies are further specified in the addendum GEP-2649, Inherited
-Policy Attachment. GEP-2649 also describes a set of expected behaviors
-for how settings can flow across a defined hierarchy.
+## TL;DR
 
+This GEP aims to standardize terminology and processes around "metaresources", i.e., using one Kubernetes object to modify the functions of one or more other objects.
 
-## Goals
+It lays out guidelines for Gateway API implementations and other stakeholders for the design and/or handling of custom resources in compliance with a pattern known as Policy Attachment.
 
-* Establish a pattern for Policy resources which will be used for any policies
-  included in the Gateway API spec
-* Establish a pattern for Policy attachment, whether Direct or Inherited,
-  which must be used for any implementation specific policies used with
-  Gateway API resources
-* Discuss the problems with communicating status for Policy objects, and suggest
-  mechanisms that Policy APIs can use to mitigate some of them.
-* Provide a way to distinguish between required and default values for all
-  policy API implementations
-* Enable policy attachment at all relevant scopes, including Gateways, Routes,
-  Backends, along with how values should flow across a hierarchy if necessary
-* Ensure the policy attachment specification is generic and forward thinking
-  enough that it could be easily adapted to other grouping mechanisms like
-  Namespaces in the future
-* Provide a means of attachment that works for both ingress and mesh
-  implementations of this API
-* Provide a consistent specification that will ensure familiarity between both
-  included and implementation-specific policies so they can both be interpreted
-  the same way.
+!!! warning
+    This GEP specifies a _pattern_, not an API field or new object. It defines some terms, including _Metaresource_, _Policies_ and _Policy Attachment_, and their related concepts.
```

Comment:
A concern here is that when I look at the [deploy preview](https://deploy-preview-3609--kubernetes-sigs-gateway-api.netlify.app/geps/gep-713/#understanding-effective-policies) for this, the number of warnings at the top of the page are pretty overwhelming. I think the text here is good, but I don't think it needs to be included in a `warning` block.

<img width="964" height="709" alt="Image" src="https://github.com/user-attachments/assets/ff7ea339-3b3d-43dd-b236-de2e22e5e515" />

Reply:
We have a `warning`, a `danger` and an `info` block here. Should them all be turned into regular paragraphs or just the `warning` one?

I'll make them all paragraphs for now, but happy to revert if needed.